packages/frontend/src/context/crabStrategy.tsx

```diff
@@ -334,7 +336,7 @@ const CrabProvider: React.FC = ({ children }) => {
   const store: CrabStrategyType = {
     maxCap,
     vault,
-    loading: loading || !ready,
+    loading: loading || !ready || userCrabBalanceLoading || currentEthLoading,
```

Comment:
It takes about 200ms to load calculate Eth. We need to handle the loading of current eth as well.